src/runtime_src/core/tools/xbmgmt2/SubCmdDump.cpp

```diff
@@ -52,6 +52,29 @@ flash_dump(const std::shared_ptr<xrt_core::device>& _dev, const std::string outp
   flasher.readBack(output);
 }
 
+static bool
+isSupported(const std::shared_ptr<xrt_core::device>& dev)
+{
+    std::string errmsg;
+    bool is_mfg = false, is_recovery = false;
+
+    try {
+      is_mfg = xrt_core::device_query<xrt_core::query::is_mfg>(dev);
+    } catch(...) {}

```

Comment:
Please don't catch (...), for query errors, use:
`const xrt_core::query::exception&`